pkg/mover/chart.go

```diff
@@ -4,18 +4,28 @@
 package mover
 
 import (
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 
 	"github.com/avast/retry-go"
 	"github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/v2/internal"
 	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/chartutil"
 )
 
 // Number of retries for pull/push operations
-const DefaultRetries = 3
+const (
+	DefaultRetries        = 3
+	EmbeddedHintsFilename = ".relok8s-images.yaml"
+)
+
+var (
+	ErrImageHintsMissing  = errors.New("no image hints provided")
+	ErrOCIRewritesMissing = errors.New("at least one rewrite rule is required")
```

Comment:
These usually require comments, isn't the editor requesting them?

Reply:
Right, my editor does not request them because my default linter is `golangci-lint`, which does not include by default golint (or revive) until this pr
https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/pull/48 is merged, 

If we merge that PR, golangci-lint configured editors will complain as well as our CI would have complained.

Reply:
Actually I was wrong. The issue is that golangci-lint has actually disabled that rule 

I have created a task to enable it (if we find it appropriate) https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes/issues/50